deployment/k8s/aws/aws-bootstrap.sh

```diff
@@ -22,9 +22,13 @@ base_dir="./opensearch-migrations"
 build_images_chart_dir="${base_dir}/deployment/k8s/charts/components/buildImages"
 ma_chart_dir="${base_dir}/deployment/k8s/charts/aggregates/migrationAssistantWithArgo"
 namespace="ma"
-skip_image_build=false
+skip_image_build=true
```

Comment:
It would be nice if there was a command line option here - or what if we just had separate scripts & maybe another script to tie these things together.